### ISSUE TITLE
CIELAB Sliders

### ISSUE THREAD

**Issue by matatk** [ThePacielloGroup/CCA-OSX#12]
This might help in making suggestions, because it should make it easier to darken/lighten a colour until a solution is found.

**Comment by ferllings** [comment]
Replace with (or add) CIELAB https://en.wikipedia.org/wiki/HSL_and_HSV#Disadvantages
See for implementation: https://github.com/THEjoezack/ColorMine/tree/master/ColorMine/ColorSpaces/Conversions

**Comment by l-i-n-k** [comment]
I agree. The RGB sliders are useless to improve an existing color pair. As a designer, I want to keep the "color" and improve with hue and/or saturation/vibrancy.

**Comment by l-i-n-k** [comment]
Thank you for the Pre-Release. Have tested the OS X native color dialogs and this helps a lot.

**Comment by ferllings** [comment]
Yes, it does help.
Now I need to implement CIELAB.
